### PR TITLE
More verbose output: Report passed tests, tool invocations.

### PR DESCRIPTION
**Pull request by garloff**
When running the test suite interactively, I want to see know what it's currently doing and what the current state of success is. This is actually already logged, but with log level debug and we have no way to make this visible by cmd line options. Rather than introducing an option `-d|--debug`, let's just attach it to `-v|--verbose`, as it does not add all that much output.

Also, when used with `-v`, let's also report all passed checks in the summary.

### CODE REVIEW

**Comment by mbuechse** [comment]
@garloff Just FYI: The tool does have `-d|--date` as well as `--debug`.

**Review by mbuechse — APPROVED** [review]
I won't stand in the way, but please consider my comment.

File: Tests/scs-compliance-check.py [hunk 1]
```diff
@@ -109,6 +109,7 @@ def apply_argv(self, argv):
                 sys.exit(0)
             elif opt[0] == "-v" or opt[0] == "--verbose":
                 self.verbose = True
+                logger.setLevel(logging.DEBUG)
```
[the remark this answers is not in the document]
Comment (garloff) [reply Tests/scs-compliance-check.py:112]: I must have overlooked the --debug setting, sorry ...

[the remark this answers is not in the document]
Comment (garloff) [reply Tests/scs-compliance-check.py]: OK, changed it to only enable `--debug` upon a second `--verbose` or `-vv`, which is in line with how many CLI tools behave.
I hope that this finds your support.

**Comment by mbuechse** [comment]
Fine by me!

**Pull request merged**